py3status/modules/netdata.py

```diff
@@ -43,19 +49,36 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 2
-    low_speed = 30
-    low_traffic = 400
-    med_speed = 60
-    med_traffic = 700
+    format = u'[\?color=down LAN(Kb): {down}↓ {up}↑] ' \
+             u'[\?color=total T(Mb): {download}↓ {upload}↑ {total}↕]'
     nic = None
+    thresholds = {
+        'down': [(0, "bad"), (30, "degraded"), (60, "good")],
+        'total': [(0, "good"), (400, "degraded"), (700, "bad")]
+    }
```

Comment:
this needs creating from the old config params if they are supplied, but get things happily working first